doc/user/content/integrations/http-api.md

```diff
@@ -0,0 +1,194 @@
+---
+title: "Connect to Materialize via HTTP"
+description: "How to use Materialize via HTTP"
+menu:
+  main:
+    parent: "integrations"
+    weight: 5
```

Comment:
This probably wants a weight that sorts it after "Overview"?

Reply:
Yeah--I clearly do not understand Hugo's weights. Ty.